gsmphys/radiation_gases.f

```diff
@@ -385,13 +385,14 @@ subroutine gas_init                                               &
             rewind NICO2CN
             read (NICO2CN, 25) iyr, cline, co2g1, co2g2
   25        format(i4,a94,f7.2,16x,f5.2)
-            co2_glb = co2g1 * 1.0e-6
+            co2_glb = co2g1 * 1.0e-6 * co2_scaling
 
             if ( ico2flg == 1 ) then
               if ( me == 0 ) then
                 print *,' - Using co2 global annual mean value from',   &
      &                  ' user provided data set:',co2usr_file
-                print *, iyr,cline(1:94),co2g1,'  GROWTH RATE =', co2g2
+                print *, iyr,cline(1:94),co2g1 * co2_scaling,           &
+     &                  '  GROWTH RATE =', co2g2
```

Comment:
Do you want to add a note that you are rescaling the data in that file by co2_scaling?